crates/css-syntax-types/src/lib.rs

```diff
@@ -45,48 +43,92 @@ impl From<&Css> for Css {
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
-#[serde(deny_unknown_fields)]
 pub struct AtRule {
```

Comment:
Given that all these types share some fields, could these be extracted to a common base type, to ensure consistency?

Reply:
Yes, but I decided against it. For consistency we would have to change a lot of structs that are outside the scope of this change. I would recommend to do that in a separate PR if really needed.